homeassistant/components/alarm_control_panel/manual_mqtt.py

```diff
@@ -86,122 +96,21 @@ class ManualMQTTAlarm(alarm.AlarmControlPanel):
     or disarm if `disarm_after_trigger` is true.
     """
 
-    def __init__(self, hass, name, code, pending_time,
-                 trigger_time, disarm_after_trigger,
-                 state_topic, command_topic, qos,
-                 payload_disarm, payload_arm_home, payload_arm_away):
+    def __init__(self, hass, name, code, pending_time, trigger_time,
+                 disarm_after_trigger, state_topic, command_topic, qos,
+                 payload_disarm, payload_arm_home, payload_arm_away,
+                 payload_arm_night, config):
         """Init the manual MQTT alarm panel."""
-        self._state = STATE_ALARM_DISARMED
-        self._hass = hass
-        self._name = name
-        self._code = str(code) if code else None
-        self._pending_time = datetime.timedelta(seconds=pending_time)
-        self._trigger_time = datetime.timedelta(seconds=trigger_time)
-        self._disarm_after_trigger = disarm_after_trigger
-        self._pre_trigger_state = self._state
-        self._state_ts = None
+        super().__init__(hass, name, code, pending_time, trigger_time,
+                         disarm_after_trigger, config)
 
         self._state_topic = state_topic
         self._command_topic = command_topic
         self._qos = qos
         self._payload_disarm = payload_disarm
         self._payload_arm_home = payload_arm_home
         self._payload_arm_away = payload_arm_away
-
-    @property
-    def should_poll(self):
```

Comment:
All this data was copied on purpose as platforms are not supposed to be depending on one another. It's fine to have duplicate code as these platforms could grow apart in the future. Please revert his change.

Reply:
I reverted and ported over the changes I made to manual alarm panel